andela/wg-leo
# [Chore #158626052] Fix coding style errors

**What does this PR do?**
Fixes the pep8 errors found in the code using flake8

**Description of task to be completed**
Many errors ensue from running flake8. These require fixing.

**How should this be manually tested?**
- Clone the repo and navigate to the project directory.
- Run `pip install flake8` to install the flake8 tool. Preferably this should be inside a virtual environment.
- Run the command `flake8` in a terminal. If no output is returned, there are no errors.

**What are relevant pivotal tracker stories?**
[#158626052](https://www.pivotaltracker.com/story/show/158626052)
